getting-started.md

```diff
@@ -21,14 +21,39 @@ To log in to the Application Service Adapter with the cf CLI:
 
     > **Note:**  If you configured the Application Service Adapter with a globally trusted certificate during installation, you can omit the `--skip-ssl-validation` flag.
 
-1. Log in with the cf CLI.
+2. Log in with the cf CLI.
 
     ```bash
     cf login
     ```
 
     The cf CLI takes you through an interactive flow to set your user name, and detects the user accounts in your local Kubeconfig file. Select one that is a cluster-admin for the Kubernetes cluster to which the Application Service Adapter is installed.
 
+3. Create Role Binding for cluster-admin user.
+
+   Use the `cf curl` command to determine your cluster-admin username
+    ```bash
+    cf curl /whoami
+    ```
+
+   Create a Role Binding for that user using the following yaml:
+   ```yaml
+   ---
+   apiVersion: rbac.authorization.k8s.io/v1
+   kind: RoleBinding
+   metadata:
+     name: cf-admin-kind-cluster
+     namespace: ROOT-NAMESPACE
```

Comment:
I don't think the root namespace is configurable in TAS Adapter. Should we just say `cf` here?